packages/utilities/psammead-rich-text-transforms/src/candy-xml.js

```diff
@@ -39,7 +39,10 @@ const createUrlLink = element => {
     blocks = [fragment(text)];
   });
 
-  return urlLink(text, locator, blocks);
+  const isInternalRegex = /.bbc.(in|com|co.uk)/;
```

Comment:
A `. ` in JS regex matches any character. `\. ` matches just the dot character. 
```suggestion
  const isInternalRegex = /\.bbc\.(in|com|co\.uk)/;
```

Reply:
Oooh good spot :O Whoops 🙈

Reply:
Great spot Sareh 😄